common/rabbitmq/CHANGELOG.md

```diff
@@ -3,13 +3,40 @@ Rabbitmq CHANGELOG
 
 This file is used to list changes made in each version of the common chart rabbitmq.
 
+
+0.6.2
+-----
+b.alkhateeb@sap.com
+- new mechanism to add customs configuration file to RabbitMQ pods, under /etc/rabbitmq/conf.d/20-custom.conf
+  custom configuration can be added as `key, value` pairs under .Values.customConfig
+  ```
+  customConfig:
+    key1: value1
+    key2: value2
+  ```
```

Comment:
Imho missing the important "keys with a null value are ignored", because that means I cannot unset a value via config, right?

Reply:
Added!